src/main/java/racing/model/PlayHistory.java

```diff
@@ -0,0 +1,22 @@
+package racing.model;
+
+import java.util.List;
+
+public class PlayHistory {
+
+    private final List<CarsAtTurn> positions;
+
+    public PlayHistory(List<CarsAtTurn> positions) {
+        this.positions = positions;
+    }
+
+    public List<CarsAtTurn> getPositions() {
+        return this.positions;
+    }
+
+    public List<String> findWinners() {
+        CarsAtTurn lastTurn = this.positions.get(this.positions.size() - 1);
```

Comment:
```suggestion
        CarsAtTurn lastTurn = lastTurn();
```

복잡한 로직을 적절한 의미를 가진 메소드로 분리시키면 어떨까요?